src/python-common/ceph/utils.py

```diff
@@ -66,3 +68,41 @@ def str_to_datetime(string: str) -> datetime.datetime:
 
     raise ValueError("Time data {} does not match one of the formats {}".format(
         string, str(fmts)))
+
+
+def parse_timedelta(delta: str) -> Optional[datetime.timedelta]:
+    """
+    Returns a timedelta object represents a duration, the difference
+    between two dates or times.
+
+    >>> parse_timedelta('foo')
+    None
```

Comment:
you're sure this is executed? I only see 

    pytest --doctest-modules ceph/deployment/service_spec.py

in the tox.ini

Reply:
Fixed